src/Service/ControllerMethodParser.php

```diff
@@ -218,4 +256,21 @@ private function parseResponses(\ReflectionMethod $method): array
 
         return $methodResponses;
     }
+
+    /**
+     * @return DataTypeParserInterface<T>
+     */
+    private function getDataTypeParser(\ReflectionProperty $reflectionProperty): DataTypeParserInterface
+    {
+        foreach (array_keys($this->dataTypeParsers->getProvidedServices()) as $key) {
+            /** @var DataTypeParserInterface<T> $dataTypeParser */
+            $dataTypeParser = $this->dataTypeParsers->get($key);
+
+            if ($dataTypeParser->supports($reflectionProperty)) {
+                return $dataTypeParser;
+            }
+        }
+
+        throw new \Exception(sprintf('Property of type %s not supported.', $reflectionProperty->getType()));
```

Comment:
Should we add a note here to let people know they could implement `\Valantic\PimcoreApiDocumentationBundle\Contract\Service\DataTypeParserInterface`?